dashboard/app/models/ability.rb

```diff
@@ -1,55 +1,48 @@
 class Ability
   include CanCan::Ability
 
+  # Define abilities for the passed in user here. For more information, see the
+  # wiki at https://github.com/ryanb/cancan/wiki/Defining-Abilities.
   def initialize(user)
-    # Define abilities for the passed in user here. For example:
-    #
-    user ||= User.new # guest user (not logged in)
-    if user.admin?
-      can :manage, :all
+    user ||= User.new
```

Comment:
As discussed offline, this is probably being done so as to eliminate the need for conjunctions like `if !user.nil? && user.admin?`. It may or may not make sense to remove this line, updating the rest of the method appropriately.